apps/api/src/app/shared/types.ts

```diff
@@ -1 +1,6 @@
 export type Constructor<I> = new (...args: any[]) => I;
+
+export type CursorPaginationParams = {
+  limit: number;
+  after: number | string;
```

Comment:
I thought about this again, it's better if we split this into two parameters creating a combo of limit, offset, and limit, after.

Reply:
which one would take the priority when both passed? `offset` or `after`?

Reply:
Throw a validation error (422) if both are passed.